### DISCUSSION TITLE
Feature request: Show battery type information for non powered devices

### DISCUSSION THREAD

**Discussion by eirik-bakke** [Koenkk/zigbee2mqtt#23737] [General]
Hi,

As the title says, it would be super useful to to have some additional data included in the information from the device database.
Most importantly would be the ability to know what kind of battery a low power device actually needs, rather than having to open it up to check. 
Along the same lines, different manufactures have different processes for resetting/repairing devices, and I find myself frequently googling to work out how a particular device needs to be treated to perform a reset.

Would it be possible to have these data added to the device database at some point in the future?

Thanks!

**Comment by vulpes2** [comment]
Reset instructions are available for some devices on zigbee2mqtt.io, so if it's not available, consider improving the documentation in that repo. As for battery information, `BatterySize` and `BatteryQuantity` are standard attributes, but most battery powered devices don't support it, and the list of batteries in the BatterySize definition contains no coin cells, which makes it pretty useless. This information will have to be manually added to zigbee2mqtt.io, and potentially included in the converters if you want the values to be displayed on the web interface.
